source/deployment/enterprise-deployment-guide.rst

```diff
@@ -0,0 +1,216 @@
+- Documentation - We link to many parts of our `administrator  documentation <https://docs.mattermost.com/guides/administrator.html>`_ in this guide but encourage you to take a look at `all our documentation <https://docs.mattermost.com>`_
+- Enterprise Support  - If you’re an Enterprise Edition subscriber, you may open a support ticket in the Enterprise Edition Support portal
+
+.. Recommendation::
+    Visit the `Mattermost Community server <https://community.mattermost.com/>`_ and create an account. Join channel ..::TODO Specific Guide Support Channel?::.. to not only experience Mattermost right away but also for support if you get stuck at any point in this guide.
+
+================================
+Before You Begin Your Deployment
+================================
+
+********************************
+What Makes Mattermost Different?
+********************************
+
+Mattermost is a high-trust messaging platform for enterprise use. As enterprises come in all shapes and sizes, Mattermost has been built to provide an optimal experience for all customers. 
+
+Taking this a step further, the emergence of ChatOps means Mattermost is no longer an exclusive developer tool. In fact, it has the ability to enhance communication through a multitude of `integrations <https://integrations.mattermost.com>`_, extensions, and customization.
+
+Here’s a sample of the types of integrations our customers are deploying:
+
+- Confluence
+- Docker
+- GitHub/GitLab
+- Jira
+- Outlook
+- Trello
+
+Taking this further, because Mattermost is an open source application, it can be customized, in many ways, to suit your organizational needs.
+
+***************
+Further Reading
+***************
+
+Mattermost Licensing
+--------------------
+
+Mattermost Enterprise Edition comes with two licensing options: Enterprise E10 and Enterprise E20 which can be tested with a `Free 30-Day Trial <https://mattermost.com/trial/>`_. The E10 license to suit smaller organizations with less need for automation and compliance, while the E20 license will provide you with features like AD/LDAP, SAML 2.0, automated compliance export, and team-specific permissions. You can find a full comparison `here <https://mattermost.com/pricing-feature-comparison>`_.
+
+
+.. Note::
+    You can purchase and manage your Mattermost Enterprise subscriptions with our `Customer Portal <https://customers.mattermost.com/login>`_ or reach out to our Sales team by contacting sales@mattermost.com.
+
+Mattermost Server Environment
+-----------------------------
+
+At its core, Mattermost is an open source, hybrid-cloud alternative to proprietary SaaS messaging for teams. It is designed to increase the agility, efficiency and innovation in high trust organizations while keeping data and operations under IT control. Mattermost ensures you can own your data. An important step is to determine the environment for the Mattermost server.
```

Comment:
@faase @justinegeffen 
We will want to rewrite that section - let's handle that in another PR.  I would recommend removing this text as it is a bit non-specific.  Recommend referencing this section in https://docs.mattermost.com/deployment/deployment.html#mattermost-server:  "The Mattermost server installs as a single compiled binary file. All server settings are stored in a configuration file, config/config.json, which can be updated directly or via a web-based System Console user interface. "  I think outlining the server dependencies and other data center requirements and pointing them to the supported versions of software/hardware are important information for them to understand at this point

Reply:
Actually - you do have a lot of this covered below. Potentially, it would be worth talking about how Mattermost server scales from teams to the enterprise.  This document does a good job describing this: https://docs.mattermost.com/overview/faq.html#how-does-mattermost-scale-from-teams-to-enterprises potentially some of this content can be reused from the perspective the server is able to scale